scripts/config/templates/nginx.conf.erb

```diff
@@ -124,7 +125,7 @@ http {
     proxy_pass https://s3.amazonaws.com/neighbor-build-${environment}/frontend$uri;
   }
 
-  location ~ ^\/sitemap.[A-Za-z_]*.xml$ {
```

Comment:
We want to start supporting numbers in our sitemaps especially since we want to start including listings (we can only have 50k urls in a sitemap and we have much more listings than that so it'd be nice to be able to have incremented names)